dataplane/linux/int_dataplane.go

```diff
@@ -1464,13 +1474,16 @@ func (d *InternalDataplane) loopUpdatingDataplane() {
 					// Dataplane is still dirty, record an error.
 					countDataplaneSyncErrors.Inc()
 				}
-				log.WithField("msecToApply", applyTime.Seconds()*1000.0).Info(
+
+				d.loopSummarizer.EndOfIteration(applyTime)
+				log.WithField("msecToApply", applyTime.Seconds()*1000.0).Debug(
 					"Finished applying updates to dataplane.")
```

Comment:
Do you still want this log as well?

Reply:
I had demoted it to debug but I've now removed it and added a clause to the summarizer so that it logs every loop if debug is enabled. Seemed like a good compromise.